Example/Common/UsingTheAnimation.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  UsingTheAnimation.swift
+//  TheAnimationExample
+//
+//  Created by Carlos on 23/5/18.
+//  Copyright © 2018 marty-suzuki. All rights reserved.
+//
+
+import QuartzCore.CoreAnimation
+import TheAnimation
+
+#if os(iOS) || os(watchOS) || os(tvOS)
+import UIKit
+public typealias Color = UIColor
+public typealias BezierPath = UIBezierPath
+#elseif os(OSX)
+import AppKit
+import CoreGraphics
+public typealias Color = NSColor
+public typealias BezierPath = NSBezierPath
+
+extension NSBezierPath {
+    var cgPath: CGPath {
```

Comment:
This is not a common property, therefore, it should not be implemented in extension.

Reply:
macOS's `NSBezierPath` hasn't `cgPath` property. What you suggest me?

Reply:
Sorry, you don't have to change about that extension👍